src/library/librarycontrol.h

```diff
@@ -1,6 +1,7 @@
 #ifndef LIBRARYMIDICONTROL_H
 #define LIBRARYMIDICONTROL_H
 
+#include <memory>
```

Comment:
Ah I forgot that was a C++14 feature... Any plans to upgrade soon? We're already using the experimental c++17 support at my work ;)